### PR TITLE
Ioffe2015batch+ishanashah

### PR DESCRIPTION
**Pull request by ishanashah**

### CODE REVIEW

**Comment by ishanashah** [comment]
@elampietti for some reason it won't let me add you as a reviewer, so I'll mention you here until I figure that out.

**Review by elampietti — COMMENTED** [review]
Nice summary, you did a good job capturing the main focus of the paper in a concise way.

I would also add some details on how the effect that batch normalization has on the gradient flow, specifically how the gradient dependency on the parameters scale is reduced, therefore a higher learning rate can be used to accelerate training without more risk of divergence. 

Another detail would be to mention the beneficial effect of generalization that batch normalization has on the model so that it does not require Dropout.
